src/Integration/TeamExplorer/ConnectSectionController.cs

```diff
@@ -31,33 +31,31 @@ namespace SonarLint.VisualStudio.Integration.TeamExplorer
     [PartCreationPolicy(CreationPolicy.Shared)]
     internal class ConnectSectionController : IServiceProvider, IDisposable
     {
-        private readonly ObservableCollection<ServerViewModel> connectedServers = new ObservableCollection<ServerViewModel>();
-        private readonly ObservableCollection<ProjectViewModel> boundProjects = new ObservableCollection<ProjectViewModel>();
         private readonly ISonarQubeServiceWrapper sonarQubeService;
         private readonly Dispatcher uiDispatcher;
         private readonly IServiceProvider serviceProvider;
         private readonly IActiveSolutionTracker solutionTacker;
 
-        private IConnectSection section;
-        private bool isDisposed = false;
-        private bool isConnecting = false;
-        private bool isBinding = false;
+        private bool isDisposed;
         private bool resetBindingWhenAttaching = true;
         private string boundSonarQubeProjectKey;
 
         [ImportingConstructor]
         public ConnectSectionController([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, SonarQubeServiceWrapper sonarQubeService, IActiveSolutionTracker solutionTacker)
-            : this(serviceProvider, sonarQubeService, solutionTacker, Dispatcher.CurrentDispatcher)
+            : this(serviceProvider, null, sonarQubeService, solutionTacker, Dispatcher.CurrentDispatcher)
```

Comment:
Rather than `null`, can we pass in `new TransferableVisualState()`, and in the internal ctor check `state` isn't null. Because `this.State` should never be null?